### ISSUE TITLE
What's the LOSS function of RRC?

### ISSUE THREAD

**Issue by KevinYuk** [xiaohaoChen/rrc_detection#28]
As is known that, the LOSS function of SSD are cross-entropy loss and  L1 smooth LOSS. How about LOSS function of RRC sub-network?

**Comment by KevinYuk** [comment]
Answered by myself. There is not LOSS function for RRC sub-network. Thanks.

**Comment by foralliance** [comment]
@KevinYuk 
i think you make a wrong conclusion．
every RRC sub-network has own LOSS function
